crates/tabby-index/src/code/types.rs

```diff
@@ -10,7 +10,7 @@ use crate::indexer::{IndexId, ToIndexId};
 #[derive(Serialize, Deserialize, Clone)]
 pub struct SourceCode {
     pub id: String,
-    pub git_url: String,
+    pub source_id: String,
```

Comment:
The refactor is to query against the index field `source_id`, there's no need to change field name to `source_id` every where.